cmd/helpMessages.go

```diff
@@ -525,21 +548,29 @@ selected file count and size:
 
    - azcopy bench --mode='Upload' "https://[account].blob.core.windows.net/[container]?<SAS>" --file-count 50000 --size-per-file 8M --put-md5
 
+Run a benchmark test that uploads 1000 files, each 100 KiB in size, and creates folders to divide up the data:
+
+   - azcopy bench "https://[account].blob.core.windows.net/[container]?<SAS>" --file-count 1000 --size-per-file 100K --number-of-folders 5
+ 
 Run a benchmark test that downloads existing files from a target
 
    - azcopy bench --mode='Download' "https://[account].blob.core.windows.net/[container]?<SAS?"
 
+Run a download benchmark with the default parameters and cap the transfer rate at 500 Mbps:
```

Comment:
why are we adding these samples ? what it a ask from any customer or something?

Reply:
these are just some more help-texts for a better coverage of features